xml/_filter.xml

```diff
@@ -148,6 +148,9 @@
     <namespaceFilter name="StackExchange.Redis">
       <typeFitler name="*" expose="false" />
     </namespaceFilter>
+    <namespaceFilter name="System.Text.Json.SourceGeneration">
+      <typeFitler name="JsonSourceGenerator" expose="false" />
```

Comment:
I changed this PR to specifically filter the `JsonSourceGenerator` class instead of the whole namespace. So we could either keep this PR, or @safern could exclude the System.Text.Json.SourceGeneration.dll assembly from the drop. Which do you think is better Eric and Santi?

Reply:
@gewarren I'll take care of the next DLL drop. The next time I send you one, I can manually remove the System.Text.Json.SourceGeneration.dll .

Reply:
My preference would be to exclude the System.Text.Json.SourceGeneration.dll assembly.

Reply:
@carlossanlop Is there a clean way of removing this assembly/namespace/type from the net6-rc1 branch? Or should we regenerate without that assembly?

Reply:
It may also be possible to automatically exclude it, but I don't know enough about how MSBuild works (Santi is the expert). We use this repo to collect the ref assemblies we send you:

https://github.com/dotnet/api-catalog-infra

There's a file that explicitly mentions the includes for the droplayout, but I don't see any excludes, and I also don't see any `System.Text.Json` there:
https://github.com/dotnet/api-catalog-infra/blob/release/6.0-rc1/eng/droplayout.props

@eerhardt maybe this file has the answer for which we are seeing `Microsoft.Extensions.Logging.Generators` and `System.Text.Json.SourceGenerators` unexpectedly?

Reply:
> Is there a clean way of removing this assembly/namespace/type from the net6-rc1 branch? Or should we regenerate without that assembly?

@gewarren I can generate the ref assemblies again, no problem. I can try to get these Source Generation assemblies explicitly excluded, or I can manually remove the DLLs. The only question I have is: if we are going to overwrite RC1 API layout, will we lose all the documentations commits we've done?

Reply:
It shouldn't overwrite if we uncheck the option to import XML doc comments.

Reply:
You don't need to generate the drop again. I'll just remove that assembly from what I have already.

Reply:
two assemblies*

Reply:
I didn't see the other assembly in the drop. But I'll look again.

Reply:
Ah apologies. I re-read @eerhardt 's other comment, and he simply mentioned that assembly was also autogenerated, but didn't say it was being included as well. It's not being included in dotnet-api-docs, I'm not seeing any files with that assembly in the net6-rc1 branch.